test/integration/dart_x_only_test.dart

```diff
@@ -0,0 +1,112 @@
+// Copyright 2015 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@TestOn('vm')
+library dart_dev.test.integration.dart_x_only_test;
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:dart_dev/util.dart' show TaskProcess;
+import 'package:test/test.dart';
+
+const String projectPath = 'test_fixtures/dart_x_only';
+
+Future<Result> runDart1Only(Iterable<String> targetArgs) =>
+    run('dart1-only', targetArgs);
+Future<Result> runDart2Only(Iterable<String> targetArgs) =>
+    run('dart2-only', targetArgs);
+
+Future<Result> run(String task, Iterable<String> targetArgs) async {
+  await Process.run('pub', ['get'], workingDirectory: projectPath);
+
+  var args = ['run', 'dart_dev', task]..addAll(targetArgs);
+  TaskProcess process =
+      new TaskProcess('pub', args, workingDirectory: projectPath);
+  final stdout = await process.stdout.join('\n');
+  print(stdout);
```

Comment:
want to leave this in here?

Reply:
Yeah, it's helpful for debugging and sanity checking that they're actually doing what we expect